src/TarStreamer.php

```diff
@@ -248,15 +248,15 @@ protected function send($data) {
 	 * Generate a PAX string
 	 *
 	 * @param array $fields key value mapping
-	 * @return string PAX formated string
+	 * @return string PAX formatted string
 	 * @link http://www.freebsd.org/cgi/man.cgi?query=tar&sektion=5&manpath=FreeBSD+8-current tar / PAX spec
 	 */
 	protected function paxGenerate($fields) {
 		$lines = '';
 		foreach ($fields as $name => $value) {
 			// build the line and the size
 			$line = ' ' . $name . '=' . $value . "\n";
-			$size = \strlen(\strlen($line)) + \strlen($line);
+			$size = \strlen((string) \strlen($line)) + \strlen($line);
```

Comment:
`strlen` needs to be passed a string - added an explicit cast otherwise phan complains.